### ISSUE TITLE
Reference Time Series:  Preview plot should exclude NoData values

### ISSUE THREAD

**Issue by horsburgh** [hydroshare/hydroshare#187]
When I put in a REST URL to retrieve a referenced time series during the resource creation process, there is a button to validate the URL and preview the time series.  A plot is created, but if there are NoData values in the time series, it isn't that interesting.  NoData values should not be plotted. See the example below

![screenshot](https://cloud.githubusercontent.com/assets/3485902/6453871/31059cb2-c100-11e4-8869-20a940754582.png)

**Comment by dtarb** [comment]
@horsburgh please test this and close if it works.

**Comment by horsburgh** [comment]
I just tested on alpha.hydroshare.org with the following REST URL:

http://data.iutahepscor.org/RedButteCreekWOF/REST/waterml_1_1.svc/datavalues?location=iutah:RB_ARBR_AA&variable=iutah:WaterTemp_EXO/methodCode=56/sourceCode=1/qualityControlLevelCode=0&startDate=&endDate=

I first got an error button stating that "This is not a valid HIS REST Url".  It seems weird that the error comes back as a clickable button.  It should probably be a label or use some kind of HydroShare standardized error reporting rather than putting the error in a clickable button that just resubmits the form.  

This error led me to check the URL that I am using in a browser window to make sure that it returned a valid WaterML file.  I discovered that my main web server was down due to an issue with our virtualization infrastructure.  So, the error that I got was not because the REST URL is invalid, but rather because HydroShare could not retrieve a WaterML file because my server was not responding.  I wonder if we could maybe do a little more descriptive error trapping - e.g., display an error that says something like "The server is not responding" rather than "This is not a valid HIS REST Url".

I got my server back up and running and then tried again. The URL validated, and the data were retrieved, but the "Invalid data" text at the bottom of the page didn't go away even though the preview plot was generated (see screen shot).

![screenshot](https://cloud.githubusercontent.com/assets/3485902/6783893/066f5d42-d142-11e4-817c-6a89711806c7.png)

Then, I clicked the "Add HIS Time Series" button at the bottom and got the following error.

![screenshot](https://cloud.githubusercontent.com/assets/3485902/6783939/3a9b0a08-d142-11e4-81eb-23c6d88d0e04.png)
